package.json

```diff
@@ -18,6 +18,11 @@
     "debug": "node-inspector & electron --debug=5858 . & xdg-open http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858",
     "doc": "jsdoc -c .jsdocrc",
     "lint": "jshint . --verbose --exclude-path=.jshintignore",
+    "postinstall": "npm run sia-repo && npm run sia-import",
+    "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
```

Comment:
I see, I'll remove the postinstall line but the other commands do no harm to stay